### DISCUSSION TITLE
4.0 release discussion

### DISCUSSION THREAD

**Discussion by pkishino** [haugene/docker-transmission-openvpn#1894] [General]
Hello all,
@haugene and me have been talking internally about what needs to be done before releasing 4.0 (mostly built on existing dev branch)
The major changes will be 
a) Move base back to Ubuntu
b) Transmission 3.0->2.94 for stability reasons
c) Configs now eternalized to https://github.com/haugene/vpn-configs-contrib, any further provider questions/issues please raise them there

**Comment by ksurl** [comment]
can you give more details on 3.0 stability issues?

**Reply by pkishino** [reply]
#1536 was one of the reason..looking through this and linked transmission issue
transmission/transmission#1314 looks like this is still an open problem

**Reply by pkishino** [reply]
I noticed this myself as well on 3.0 and when downgrading to 2.94 the problems dissapeared

**Comment by braymullo** [comment]
Looking forward to trying it out :) thanks all for the hard work
